editor/components/post-permalink/index.js

```diff
@@ -72,8 +168,14 @@ export default connect(
 	( state ) => {
 		return {
 			isNew: isEditedPostNew( state ),
-			link: getEditedPostAttribute( state, 'link' ),
+			previewLink: getEditedPostPreviewLink( state ),
+			samplePermalinkData: getEditedPostAttribute( state, 'sample_permalink' ),
```

Comment:
Why do we suffix as "Data" ? It leads me to wonder if it's some complex data shape, but it's just a string. I might guess it's to avoid conflicting with the components state naming, but aside from notes above to consider not assigning these values into state, we should also consider that props are the public-facing interface that developers interact with, and should take precedence over whatever implementation details need to occur within the component itself.

Reply:
Well, it's an array with two strings in it. 🙂

I'm fine with changing the name to something more appropriate, or moving it out of the props, but my familiarity with the correct way to do that is... minimal.